common/app/common/models/AttributeBean.java

```diff
@@ -0,0 +1,62 @@
+package common.models;
+
+import common.contexts.UserContext;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.products.attributes.Attribute;
+import io.sphere.sdk.products.attributes.AttributeAccess;
+import io.sphere.sdk.products.attributes.AttributeExtraction;
+import io.sphere.sdk.producttypes.MetaProductType;
+
+public class AttributeBean extends Base {
+    private String name;
+    private String key;
+    private String value;
+
+    public AttributeBean() {
+    }
+
+    public AttributeBean(final Attribute attribute, final MetaProductType metaProductType, final UserContext userContext) {
+        this.name = metaProductType
+                .findAttribute(attribute.getName())
+                .map(def -> def.getLabel().find(userContext.locales()).orElse(""))
+                .orElse(null);
+        this.key = attribute.getName();
+        this.value = formatValue(metaProductType, attribute, userContext);
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public void setKey(final String key) {
+        this.key = key;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(final String value) {
+        this.value = value;
+    }
+
+    protected static String formatValue(final MetaProductType metaProductType, final Attribute attribute,
+                                        final UserContext userContext) {
+        final AttributeExtraction<String> attributeExtraction = AttributeExtraction.of(metaProductType, attribute);
+        return attributeExtraction
+                .ifIs(AttributeAccess.ofLocalizedString(), v -> v.find(userContext.locales()).orElse(""))
+                .ifIs(AttributeAccess.ofLocalizedEnumValue(), v -> v.getLabel().find(userContext.locales()).orElse(""))
```

Comment:
I started using this method, but I rollbacked for these reasons:
1) it is unlikely that you have `en-UK` and `en` defined in your project, you would usually have `en-UK` and `en-US` in any case, and this is not helpful for the method.
2) we already use several fallbacks to make sure we get some language (user-agent preferred languages, project default language)

Of course these are not reasons to stop using this method, because these are just arguments to evidence that is not a great help in this case. The real reason is that it will pollute the code with `Optional.ofNullable(v.getLabel().getTranslation(userContext.locales())).orElse("")` which is not so nice anymore and again we require some util class to support it (which anyway is not a bad idea, as it is easier to change the way of getting the translations).

Anyway, not a topic for this PR, but I'll probably try some idea in future PR.